shard-operator-parent/shard-operator-v1/src/test/filtered-resources/reportportal.properties

```diff
@@ -0,0 +1,6 @@
+rp.endpoint = ${rp.endpoint}
+rp.uuid = ${rp.uuid}
+rp.launch = ${rp.launch}
+rp.project = ${rp.project}
+rp.attributes = ${rp.attributes}
+rp.enable = ${rp.enable}
```

Comment:
@jiripetrlik 
FYI
I don't expect this change to have impact on ReportPortal configuration